src/cf-core/src/cf-base.less

```diff
@@ -336,7 +336,7 @@ h6,
     margin-bottom: unit( 15px / 18px, em );
 
     .respond-to-max(@bp-xs-max, {
-        // Use the same regular weight but reduce the sizes to h4 size
```

Comment:
Is this removed because it's violating a stylelint rule? I think we still need this comment, in whatever form stylelint will allow.